xarray/tests/test_conventions.py

```diff
@@ -63,7 +63,13 @@ def test_decode_cf_with_conflicting_fill_missing_value() -> None:
         np.arange(10),
         {"units": "foobar", "missing_value": np.nan, "_FillValue": np.nan},
     )
-    actual = conventions.decode_cf_variable("t", var)
+
+    # the following code issues two warnings, so we need to check for both
```

Comment:
@dcherian I've tried to make this way shorter. Using for-loop looks ugly, though.